docs/pages/x/api/data-grid/data-grid.js

```diff
@@ -1,5 +1,5 @@
 import * as React from 'react';
```

Comment:
The new page looks really nice!

We can improve the function props though, here is an example:

<img width="652" src="https://github.com/mui/mui-x/assets/13808724/a3ed1e95-08f0-44a0-ab55-85bb5231f711">

---

The description below the function signature is confusing, here's a jsdoc of that prop:

<img width="574" src="https://github.com/mui/mui-x/assets/13808724/85383ad3-66ed-4c66-a4d9-95634a7148c1">

---

We definitely miss arguments names here, as well as the "Returns" label, something like this:

<img width="625" src="https://github.com/mui/mui-x/assets/13808724/2e3196ac-b043-429a-a01e-63e8b62a60ab">

---

What do you think?

Reply:
Nice spot, I forget those details. It should be fixed now. I had to refresh a lot the deploy before seeing the new page

https://deploy-preview-9187--material-ui-x.netlify.app/x/api/data-grid/data-grid/#prop-getCellClassName

![image](https://github.com/mui/mui-x/assets/45398769/c9d0bd99-0e45-4d47-86f8-cd64db854c64)

Reply:
Is this shift to the right intentional here?

<img width="607" src="https://github.com/mui/mui-x/assets/13808724/96802553-0527-47ce-bfdd-0f0385cf9410">

Reply:
You're talking about the bullet points/return?

For them yes it's expected (designed refinement coming from [this feedback](https://github.com/mui/material-ui/pull/37405#issuecomment-1628373858))

Here is an example with [Autocomplete filterOptions](https://mui.com/material-ui/api/autocomplete/#prop-filterOptions)

![image](https://github.com/mui/mui-x/assets/45398769/7f3628df-9f06-4cbb-88da-0347a4315fce)